nuxeo-runtime/nuxeo-runtime-management/src/main/java/org/nuxeo/runtime/management/api/ProbeStatus.java

```diff
@@ -125,4 +126,11 @@ public String getAsXML() {
         return sb.toString();
     }
 
+    /**
+     * @since 11.1
+     */
+    public Map<String, String> getInfos() {
+        return Collections.unmodifiableMap(infos);
```

Comment:
Not sure why you need an `unmodifiableMap`.

Reply:
it was suggested to me by @tmartins to make it read-only. It's more efficient than copying the map so it's not an extra but lower cost. And in that specific case why would we need it to be modifiable ?

Reply:
If you want to make it read only it's definitely the right approach.
Question is not why we need it to be modifiable by why the contrary, as it's just a POJO?
Anyway, can't hurt!